build.gradle

```diff
@@ -6,18 +6,18 @@ defaultTasks 'clean', 'libs'
 sourceSets {
     jobs {
         groovy {
-            srcDir "platform/jobs"
-            srcDir "sample/jobs"
+            srcDir 'platform/jobs'
+            srcDir 'sample/jobs'
         }
```

Comment:
Trying to standardize... but unfortunately some of the strings are GStrings the repo as a whole to match codenarc style